hw3/app/app/src/Search.jsx

```diff
@@ -0,0 +1,56 @@
+import { useState } from 'react';
+
+function Search(props) {
+  const { title } = props;
+  const { characters } = props;
+  const [input, setInput] = useState('');
+  const [search, setSearch] = useState('');
+
+  const handleChange = (event) => {
+    setInput(event.target.value);
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    setSearch(input.toLowerCase());
+  };
+
+  function executeSearch() {
+    let results = characters.filter((character) => {
+      if (character.fullName.toLowerCase().includes(search)) {
+        return character;
+      }
+      return '';
+    });
+    return results.map((character) => (
+      <div className='d-flex flex-column justify-content-center align-items-center m-2'>
+        <img
+          src={character.imageUrl}
+          alt='a GOT character'
```

Comment:
Duplicate alternative text aren't going to be very helpful for those using accessibility tools. You might consider updating this alt to include the full name of the character to be more descriptive.